### DISCUSSION TITLE
28. March 2023

### DISCUSSION THREAD

**Discussion by dimakuv** [gramineproject/gramine#1244] [Meeting notes]
### Agenda

*(please write your proposed agenda items in comments under this discussion)*

- [ ] ~moved from 2 weeks ago: gRPC (Leila)~
- [x] moved from last week: musl (Anjali/Anees)

### Misc

Bobby Marinov: from Fortanix.

### Woju: musl

Anjali/Anees were trying to run musl binaries on RHEL. What can be done: take unmodified binaries, copy it to the RHEL machine, and just execute. What can't be done: take the sources (linked against Musl) and compile on RHEL.

Anjali/Sankar: When Gramine is built with `-Dtests=enabled -Dmusl=enabled`, then the build breaks.

### Dmitrii: do we have an out-of-cycle release?

Because of this CVE fix: https://github.com/gramineproject/gramine/pull/1237

Michal/Mona: probably not, because this CVE has a medium CVE score of 5.5 (https://nvd.nist.gov/vuln/detail/CVE-2022-21166).
- Mona: let's not do an out-of-cycle release.

**Comment by woju** [comment]
- [ ] moved from 2 weeks ago: gRPC (Leila)
- [ ] moved from last week: musl (Anjali/Anees)
